debian/clearwater-snmp-alarm-agent.upstart

```diff
@@ -1,37 +1,13 @@
-# clearwater-socket-factory
-#
-# This process runs as root in the default namespace, and enables processes
-# running in different namespaces to obtain sockets in other namespaces.
-
 description "Clearwater alarm agent"
 
 start on (starting clearwater-monit)
 stop on runlevel [!2345]
 
 respawn
-# Sleep for 5 seconds on stop, to avoid respawning too quickly
-post-stop exec sleep 5
```

Comment:
I deleted this and moved the code into the script block below (for consistency with other scripts). Though in retrospect this way seems better (since you sleep after the process has exited, rather than before it starts). I'll migrate the other script I saw to use this approach.